src/test/java/dev/jbang/util/TestCommandBuffer.java

```diff
@@ -45,16 +45,18 @@ void testRunWinPS1() {
 	}
 
 	@Test
+	@EnabledOnOs(OS.WINDOWS)
 	void testApplyWindowsMaxLengthLimitExe() throws IOException {
 		ProcessBuilder pb = CommandBuffer.of(argsTooLong("foo.exe"))
 			.shell(Util.Shell.cmd)
 			.applyWindowsMaxLengthLimit()
 			.asProcessBuilder();
-		assertThat(pb.command().size(), greaterThan(2));
-		assertThat(pb.command().get(1), not(anyOf(startsWith("@"), startsWith("\"@"))));
+		assertThat(pb.command().size(), equalTo(2));
+		assertThat(pb.command().get(1), anyOf(startsWith("@"), startsWith("\"@")));
 	}
 
 	@Test
+	@EnabledOnOs(OS.WINDOWS)
```

Comment:
This limit is not necessary, the test will work on any platform, even if it's meant for Windows

Reply:
Ah, but of course with your changes, it _is_ necessary. Forget I said anything :-)